### ISSUE TITLE
Redirect Request - Find a form detail page for 21-0781a

### ISSUE THREAD

**Issue by mnorthuis** [department-of-veterans-affairs/va.gov-team#86584]
### Instructions
- Requests must be submitted **AT LEAST** 2 weeks in advance.  Some requests take a significant amount of time and coordination to implement, so start the process as soon as you know you will need one. 
- This issue will be used from initial request through implementation to ensure all individuals working on this are notified of status updates.  Please do not create multiple issues to track differnt steps.
- It is your responsibility to notify VA stakeholders as appropriate.  

| POC | Github alias(es) |
--- | --- 
| Product team point(s) of contact: | @shannonkford  |
| Content point of contact: | @bethpottsVADEPO   |
| IA point of contact: | @mnorthuis  |

### Type of request
- [x] We are retiring or taking down a page and need to redirect the URL (complete redirect section)
- [ ] We are changing the URL of an existing page (complete redirect section)
- [ ] We need a custom vanity URL (complete vanity URL section)
- [ ] We are removing a temporary redirect  (add relevant background and link to original redirect request)

### Do you need Public Websites to implement this redirect?
- [x] Yes
- [ ] No - Please note: Redirect still needs approval from IA and PW will do PR review

### Implementation date
When does this request need to be live? **NOTE:** Redirects deploy M-Th. Friday launches should be avoided.
Choose one:
- [x] On a specific date, connected to a program or publishing requirement. **06/27/2024**
- [ ] Any day prior to a deadline/latest possible date. **DEADLINE:** 
- [ ] Any day after an earliest possible date. **EARLIEST DATE:**
- [ ] During a specific date range: ***EARLIEST DATE TO LATEST DATE:***
- [ ] No hard timing requirements, publish when possible.

### Redirects 
Current URL  |  Redirect Destination or New URL
---  |  ---
https://www.va.gov/find-forms/about-form-21-0781a/ | https://www.va.gov/find-forms/about-form-21-0781/

The 21-0781a form will be combined with the 21-0781 form, and therefore removed from the forms db on 6/28/2024.  

### Vanity URLs 
Vanity URL  |  Landing page
---  |  ---
requested URL | landing page URL

**Link to campaign landing page request issue:**

### Process, Roles and Responsibilities
- [x] Requesting team: Above information is provided
- [x] Requesting team: All appropriate VA stakeholders are notified as appropriate
- [x] IA: Request is vetted and documented and implementation plan is clear 
- [x] IA: Request is assigned to appropriate team for implementation 
- [ ] Implementation team: Work is complete
- [ ] Implementation team: Validated in production
- [ ] Requesting team: Validates everything is correct in production and closes ticket

**Comment by mnorthuis** [comment]
This request is vetted and stakeholders are aware of the request, so this can move forward to implementation. 

The form will be removed on Friday, June 28th, it is ok to implement this on Thursday, June 27th.

**Comment by shannonkford** [comment]
@mnorthuis The content doc provided by VBA seems to indicate a desire to leave the about 0781a page intact and explain that it has been retired - wondering a) what y'all think about that and b) if we need a redirect on the 0781a PDF file itself.

**Comment by jilladams** [comment]
(Verified Mikki's suspicion: when the Forms DB marks a form as Deleted, Drupal auto-archives the related form page. )

**Comment by jilladams** [comment]
@FranECross since you're getting sick and I'll forget, I went ahead and tagged into Sprint 7 so we don't lose track of this.

**Comment by jilladams** [comment]
Is this already IA approved, since it came from Mikki? Or do @jennymayoco @kristinoletmuskat still need to take a look?

**Comment by mnorthuis** [comment]
@jilladams I vetted this one when the request came in, so you can move forward.

**Comment by jilladams** [comment]
SCHEDULE: 
* Wed 6/26: Redirect must merge after Devops daily deploy, for Thurs deploy
* Thurs 6/27, 10a ET: Redirect should deploy 
* Fri 6/28: The form will get marked deleted in the Forms DB 
* Overnight (Fri night into Sat? or into Mon? Not sure): that fact will get migrated into Drupal during the night, and the page will get auto-archived.

Also noting: Devops does not auto-deploy on Fridays, so if we miss the Thurs 6/27 daily deploy, we'll need to request an OOB deploy so the page doesn't 404 between Fri/Mon when it's removed from the Forms DB.

**Comment by randimays** [comment]
Pulling in as stretch for Sprint 6.

**Comment by jilladams** [comment]
Temporarily adding Katherine Fung as an assignee so she'll get updates when this deploys / we verify in prod.

**Comment by randimays** [comment]
@kristinoletmuskat I have verified this redirect is working in production. Could you validate on your end and close this ticket if so?

**Comment by kristinoletmuskat** [comment]
Verified it works! Closing ticket

**Comment by jilladams** [comment]
@bethpottsVADEPO the broken link checker is reporting a few pages that still link to the old URL vs. the redirected one: 

Source: https://prod.cms.va.gov/disability/how-to-file-claim/additional-forms
Broken link: https://va.gov/find-forms/about-form-21-0781a
<a data-entity-substitution="canonical" data-entity-type="node" data-entity-uuid="b4da777c-6c3e-4bf4-8ea3-d91239b15c47" href="/find-forms/about-form-21-0781a" title="About VA Form 21-0781a">Get VA Form 21-0781a to download</a>

Source: https://prod.cms.va.gov/disability/how-to-file-claim/evidence-needed
Broken link: https://va.gov/find-forms/about-form-21-0781a
<a href="/find-forms/about-form-21-0781a" data-entity-type="node" data-entity-uuid="b4da777c-6c3e-4bf4-8ea3-d91239b15c47" data-entity-substitution="canonical" title="About VA Form 21-0781a">Get VA Form 21-0781a to download</a>

Source: https://prod.cms.va.gov/disability/eligibility/ptsd
Broken link: https://va.gov/find-forms/about-form-21-0781a
<a data-entity-substitution="canonical" data-entity-type="node" data-entity-uuid="b4da777c-6c3e-4bf4-8ea3-d91239b15c47" href="/find-forms/about-form-21-0781a" title="About VA Form 21-0781a">Get VA Form 21-0781a&#xA0;to download</a>

**Comment by shannonkford** [comment]
Thanks @jilladams We know about two of these, @katherine-fung I missed the evidence needed page. The plan is to update these pages next.

**Comment by bethpottsVADEPO** [comment]
@RLHecht is taking care of this.

**Comment by RLHecht** [comment]
Just commented in slack but I'm holding off publishing because i saw @katherine-fung has draft changes on both pages and I don't know if they can be published. Since these are currently redirecting and now showing broken links I think it's OK for Katherine to do when she's back. Thanks!

**Comment by katherine-fung** [comment]
Hi @shannonkford @jilladams @RLHecht @bethpottsVADEPO, 

Randi is correct - I've already made draft changes to these 3 static pages:

1. https://prod.cms.va.gov/disability/how-to-file-claim/additional-forms
2. https://prod.cms.va.gov/disability/how-to-file-claim/evidence-needed
3. https://prod.cms.va.gov/disability/eligibility/ptsd

The plan is to copy edit and publish these pages this week. I've also confirmed that the links to the 0781a page are re-directing and not showing broken links.
